### PR TITLE
Fixed CSV import

### PR DESCRIPTION
**Pull request by jsangmeister**
Replaced all ViewModel classes with Model classes in import
Added username and gender to user import and export and adjusted example file
Renamed `AgendaImport*` to `TopicImport*` to resemble the function more appropriately

### CODE REVIEW

File: client/src/app/core/repositories/users/user-repository.service.ts [hunk 1]
```diff
@@ -209,10 +209,11 @@ export class UserRepositoryService extends BaseRepository<ViewUser, User, UserTi
      *
      * @param newEntries
      */
-    public async bulkCreate(newEntries: NewEntry<ViewUser>[]): Promise<number[]> {
+    public async bulkCreate(newEntries: NewEntry<User>[]): Promise<number[]> {
         const data = newEntries.map(entry => {
-            return { ...entry.newEntry.user, importTrackId: entry.importTrackId };
+            return { ...entry.newEntry, importTrackId: entry.importTrackId };
         });
+        console.log(newEntries, data);
```
Comment (FinnStutzenstein) [inline client/src/app/core/repositories/users/user-repository.service.ts]: Plase remove

File: client/src/app/core/ui-services/base-import.service.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ type CsvImportStatus = 'new' | 'error' | 'done';
 @Injectable({
     providedIn: 'root'
 })
-export abstract class BaseImportService<V extends BaseViewModel> {
+export abstract class BaseImportService<V extends BaseModel> {
```
Comment (FinnStutzenstein) [inline client/src/app/core/ui-services/base-import.service.ts]: Please rename generic parameter `V` to `M`.

File: client/src/app/site/base/base-import-list.ts [hunk 3]
```diff
@@ -8,11 +8,11 @@ import { TranslateService } from '@ngx-translate/core';
 import { auditTime } from 'rxjs/operators';
 
 import { BaseImportService, NewEntry, ValueLabelCombination } from 'app/core/ui-services/base-import.service';
+import { BaseModel } from 'app/shared/models/base/base-model';
 import { getLongPreview, getShortPreview } from 'app/shared/utils/previewStrings';
 import { BaseViewComponent } from './base-view';
-import { BaseViewModel } from './base-view-model';
 
-export abstract class BaseImportListComponent<V extends BaseViewModel> extends BaseViewComponent implements OnInit {
+export abstract class BaseImportListComponent<V extends BaseModel> extends BaseViewComponent implements OnInit {
```
Comment (FinnStutzenstein) [inline client/src/app/site/base/base-import-list.ts]: Same as above.

File: openslides/users/views.py [hunk 4]
```diff
@@ -368,6 +369,7 @@ def mass_import(self, request):
         inform_changed_data(created_users)
         return Response(
             {
+                "users": users,
```
Comment (FinnStutzenstein) [inline openslides/users/views.py]: Where do you use this value in the client?

File: client/src/app/site/agenda/services/agenda-import.service.ts [hunk 5]
```diff
@@ -8,18 +8,17 @@ import { TopicRepositoryService } from 'app/core/repositories/topics/topic-repos
...
 import { DurationService } from 'app/core/ui-services/duration.service';
 import { ItemVisibilityChoices } from 'app/shared/models/agenda/item';
-import { ViewCreateTopic } from 'app/site/topics/models/view-create-topic';
 import { CreateTopic } from '../../topics/models/create-topic';
 
 @Injectable({
     providedIn: 'root'
 })
-export class AgendaImportService extends BaseImportService<ViewCreateTopic> {
+export class AgendaImportService extends BaseImportService<CreateTopic> {
```
Comment (tsiegleauq) [inline client/src/app/site/agenda/services/agenda-import.service.ts]: Please rename that class. This service imports topics, not an agenda. Importing an agenda would be a tremendous amount of unclear tasks

**Pull request merged**